inc/block-editor-panels/register-meta.php

```
@@ -0,0 +1,90 @@
+	foreach ( $post_types as $post_type ) {
+		foreach ( $checkbox_metas as $meta_key ) {
+			register_post_meta(
+				$post_type,
+				$meta_key,
+				array(
+					'type'              => 'string',
+					'single'            => true,
+					'sanitize_callback' => 'sanitize_text_field',
+					'show_in_rest'      => true,
+					'auth_callback'     => function () {
+						return current_user_can( 'edit_posts' ); },
```

PHPStan (2.1.46)

[error] Function current_user_can not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)